thundra/handler.py

```diff
@@ -0,0 +1,15 @@
+import os
+from importlib import import_module
+from thundra.thundra_agent import Thundra
+
+thundra = Thundra()
+handler_path = os.environ.get('thundra_agent_lambda_handler', 'app.handler')
```

Comment:
There should not be default value. If the env variable is not set, it should fail with a proper error message.